### PR TITLE
Renaming azure_rm_managed_disk for consistency

### PR DESCRIPTION
**Pull request by zikalino**
##### SUMMARY
This change removes extra underscore in 2 module names

##### ISSUE TYPE
- Bugfix Pull Request

##### COMPONENT NAME
azure_rm_manageddisk
azure-rm_manageddisk_facts

### CODE REVIEW

File: lib/ansible/config/module_defaults.yml [hunk 1]
```diff
@@ -524,9 +524,9 @@ groupings:
   - azure
   azure_rm_loadbalancer_facts:
   - azure
-  azure_rm_managed_disk:
+  azure_rm_manageddisk:
```
Comment (yungezz) [inline lib/ansible/config/module_defaults.yml:527]: what 's  this file used for?

Comment (zikalino) [reply lib/ansible/config/module_defaults.yml:527]: Seems like the purpose of this file is to assign modules to groups.

And then user could use **module_defaults** like described here https://docs.ansible.com/ansible/latest/user_guide/playbooks_module_defaults.html
on **group/azure** for all azure modules, like this:

- hosts: localhost
  module_defaults:
    **group/azure**:
      location: eastus
  tasks:
    - azure_rm_xxxx:

But that doesn't seem to be very common feature, and I think it's not maintained very well.

**Review by yungezz — APPROVED** [review]

**Comment by jeking3** [comment]
You changed the name of azure_rm_managed_disk to azure_rm_manageddisk?  Isn't that a breaking change?  Every playbook that uses one of these is going to fail?

**Comment by mattclay** [comment]
@jeking3 Aliases were added for the old module names, so they will continue to work.

**Comment by jeking3** [comment]
Okay, phew!  Thanks, I missed that in my panic. :)

**Pull request merged**